File: app/src/main/java/ml/dvnlabs/animize/recyclerview/interfaces/addingQueue.java
package ml.dvnlabs.animize.recyclerview.interfaces;

import ml.dvnlabs.animize.recyclerview.packagelist.starlist_adapter;

import java.util.ArrayList;

public interface addingQueue {
    void addQueue(String pkganim,int position);
    void removeQueue(int position);
    ArrayList<starlist_adapter.requestQueue> getQueue();
}

File: app/src/main/java/ml/dvnlabs/animize/ui/recyclerview/interfaces/AddingQueue.kt
/*
 * Copyright (c) 2020.
 * Animize Devs
 * Copyright 2019 - 2020
 * Davin Alfarizky Putra Basudewa <dbasudewa@gmail.com>
 * This program used for watching anime without ads.
 *
 */
package ml.dvnlabs.animize.ui.recyclerview.interfaces

import ml.dvnlabs.animize.ui.recyclerview.packagelist.StarListAdapter
import java.util.*

interface AddingQueue {
    fun addQueue(pkganim: String, position: Int)
    fun removeQueue(position: Int)
    val queue: ArrayList<StarListAdapter.requestQueue?>?
}